src/main/ruby/core/truffle/ffi/ffi_struct.rb

```diff
@@ -266,6 +271,12 @@ def size
     def members
       self.class.members
     end
+    
+    def to_h
+      members.map { |member|
+        [member, self[member]]
+      }.to_h
```

Comment:
We'd generally use `do`...`end` here as well. But I admit calling `end.to_h` looks funny so I personally don't mind the use of curly braces here.